cache-config/t3cutil/t3cutil.go

```diff
@@ -81,7 +81,7 @@ func NewLineFilter(str string) string {
 func ReadFile(f string) []byte {
 	data, err := ioutil.ReadFile(f)
 	if err != nil {
-		fmt.Println("Error reading file ", f)
+		log.Errorln("Error reading file ", f)
```

Comment:
Uhh...libraries should never log. We can't say whether loggers have been initialized, but if they haven't, someone is going to end up with their app exiting/crashing and it's going to be extremely confusing to figure out why.

This function isn't used anywhere. Please just delete the function, if you don't mind.

Reply:
removed unused func